test/functional/pdo_sqlsrv/skipif_version_less_than_2k16.inc

```diff
@@ -1,6 +1,4 @@
 <?php
-if ( !( strtoupper( substr( php_uname( 's' ),0,3 ) ) === 'WIN' ) ) die( "Skip Test on windows only." ); 
-
```

Comment:
Does this one and the equivalent one for 2k14 need to check ODBC version? only works for 17.2 right?